### PR TITLE
5 search and choose the best way to connect the mongodb database with fastapi and use it

### LINKED ISSUE #5: Search and choose the best way to connect the MongoDB database with FastAPI and use it.

### PR DESCRIPTION
**Pull request by iyassalah**
@yamenw please review these changes.

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+
+class Hyperparameter(BaseModel):
+    """Hyperparameter class"""
+    created: datetime
+    params: object
+    fp: int
+    tp: int
+    fn: int
+    tn: int
+
```
Comment (yamenw) [inline app/models.py:22]: Please consider moving these to a different file, called `schema.py`, this ensures that API data models and database schema models aren't mixed up.

File: app/routers/abstract.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+"""Router for abstact"""
+
+from bson import ObjectId
+from fastapi import APIRouter, HTTPException
+from ..database import db
+from ..models import Abstract
+
+router = APIRouter(prefix="/Abstract")
```
Comment (yamenw) [inline app/routers/abstract.py:8]: Please avoid using uppercase letters in endpoints, it is error prone.

File: app/database.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+'''Any type'''
+from typing import Any
+from pymongo import MongoClient
+from .settings import MONGODB_URL
+
+client: Any  = MongoClient(MONGODB_URL)
```
Comment (yamenw) [inline app/database.py]: Please avoid using the `Any` type, use an appropriate datatype.

**Review by yamenw — APPROVED** [review]
I have approved this, but please reflect these changes _before_ merging, thanks for working on this.

**Comment by iyassalah** [comment]
@yamenw thank you for this review :)
all notes was fixed, and I am going to merge now.

**Pull request merged**